### PR TITLE
Add initial AWS images (ubuntu 18.04, windows 2019, windows 2019 + docker)

### PR DESCRIPTION
**Pull request by slide**

### CODE REVIEW

File: ubuntu-18-agent-aws.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      "tags": {
+        "timestamp": "{{isotime \"20060102150405\"}}",
+        "imagetype": "jenkins-agent-ubuntu18.04"
+      }
+    }
+  ],
+  "provisioners": [{
+    "execute_command": "chmod +x {{ .Path }}; {{ .Vars }} sudo -E sh '{{ .Path }}'",
+    "inline": [
+      "useradd -m -o -u 1000 jenkins",
```
Comment (olblak) [inline ubuntu-18-agent-aws.json]: Isn't already used by the ubuntu user?

Comment (slide) [reply ubuntu-18-agent-aws.json]: Yes, the `-o` means that the UID doesn't need to be unique.

**Comment by olblak** [comment]
The Jenkinsfile also needs to be refactored in order to be able to push images to AWS, I propose to create two directories aws and azure and move packer files in their respective directories

**Comment by timja** [comment]
> The Jenkinsfile also needs to be refactored in order to be able to push images to AWS, I propose to create two directories aws and azure and move packer files in their respective directories

makes sense

File: ubuntu-18-agent-aws.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+      "tags": {
+        "timestamp": "{{isotime \"20060102150405\"}}",
+        "imagetype": "jenkins-agent-ubuntu18.04"
+      }
+    }
+  ],
+  "provisioners": [{
+    "execute_command": "chmod +x {{ .Path }}; {{ .Vars }} sudo -E sh '{{ .Path }}'",
+    "inline": [
```
Comment (timja) [inline ubuntu-18-agent-aws.json]: is is worth extracting this out to a script? I don't see any changes between aws and azure here?

Comment (olblak) [reply ubuntu-18-agent-aws.json]: It would a nice improvement but not mandatory IMHO.

Comment (timja) [reply ubuntu-18-agent-aws.json]: should be straightforward since windows is done like this

**Comment by slide** [comment]
> The Jenkinsfile also needs to be refactored in order to be able to push images to AWS, I propose to create two directories aws and azure and move packer files in their respective directories

I started on updating the Jenkinsfile, but didn't want to push anything without the credentials in there. I can change it to use directories for sure.

File: ubuntu-18-agent-aws.json (hunk 2 above)
Comment (slide) [reply ubuntu-18-agent-aws.json]: Should be pretty easy, I can pull it out as part of this.

**Comment by timja** [comment]
> > The Jenkinsfile also needs to be refactored in order to be able to push images to AWS, I propose to create two directories aws and azure and move packer files in their respective directories
> 
> I started on updating the Jenkinsfile, but didn't want to push anything without the credentials in there. I can change it to use directories for sure.

creds should be there, packer-aws

**Comment by olblak** [comment]
> packer

I replace the aws credentials by `packer-aws-secret-access-key` and `packer-aws-secret-access-key`

**Pull request merged**